### PR TITLE
basic-cli build

### PR DESCRIPTION
**Pull request by Anton-4**
Sidenote: Files for windows are not yet included in the platform archive because we don't have nightly releases for windows yet.

### CODE REVIEW

File: .github/workflows/benchmarks.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-on: [pull_request]
+on: #[pull_request]
```
Comment (rtfeldman) [inline .github/workflows/benchmarks.yml:1]: Is commenting this out intentional?

File: ci/get_latest_release_url.sh [hunk 2]
```diff
@@ -1,16 +1,16 @@
-#!/usr/bin/env bash
+ #!/usr/bin/env bash
```
Comment (rtfeldman) [inline ci/get_latest_release_url.sh:1]: Is the added space here intentional?

Comment (Anton-4) [reply ci/get_latest_release_url.sh:1]: No it is not :) I'll remove it.

**Comment by Anton-4** [comment]
ping @rtfeldman

**Pull request merged**